supla-common/proto.h

```diff
@@ -680,6 +727,48 @@ typedef struct {
   unsigned _supla_int_t disablesLocalOperation;
 } TActionTriggerProperties;
 
+#define SUPLA_HVAC_VALUE_FLAG_SETPOINT_TEMP_MIN_SET (1ULL << 0)
+#define SUPLA_HVAC_VALUE_FLAG_SETPOINT_TEMP_MAX_SET (1ULL << 1)
+// Tells if output responsible for heating function is enabled
+#define SUPLA_HVAC_VALUE_FLAG_HEATING (1ULL << 2)
+// Tells if output responsible for cooling function is enabled
```

Comment:
Is it enabled or available?

Reply:
it tells that there is relay enabled, so we are actually heating now